test/functional/helpers.lua

```diff
@@ -45,6 +45,8 @@ module.nvim_argv = {
   '--cmd', 'unmap <C-L>',
   '--cmd', 'iunmap <C-U>',
   '--cmd', 'iunmap <C-W>',
+  '--cmd', 'xunmap *',
+  '--cmd', 'xunmap #',
```

Comment:
could mash these unmaps into a single `|` separated `--cmd`, to avoid running out of `--cmd` (max 10 or so).

Reply:
I tried to fix it, but is it correct?
Test fails with timeout